tests/seed_isort_config_test.py

```diff
@@ -95,12 +99,72 @@ def test_integration_extra_file(tmpdir):
         assert tmpdir.join('.isort.cfg').read() == expected
 
 
-def test_integration_no_config(tmpdir, capsys):
+@pytest.mark.parametrize(
+    'initial_filesystem,expected_filesystem',
+    [
+        ({}, {'.isort.cfg': '[settings]\nknown_third_party=cfgv\n'}),
+        (
+            {'setup.cfg': '[bdist_wheel]\nuniversal = True\n'},
+            {
+                'setup.cfg': '[bdist_wheel]\nuniversal = True\n\n'
+                '[isort]\nknown_third_party=cfgv\n',
+            },
+        ),
+        (
+            {'tox.ini': '[tox]\nenvlist = py36\n'},
+            {
+                'tox.ini': (
+                    '[tox]\nenvlist = py36\n\n'
+                    '[isort]\nknown_third_party=cfgv\n'
+                ),
+            },
+        ),
+        (
+            {
+                'setup.cfg': '[bdist_wheel]\nuniversal = True\n',
+                'tox.ini': '[tox]\nenvlist = py36\n',
+            },
+            {
+                'setup.cfg': '[bdist_wheel]\nuniversal = True\n',
+                'tox.ini': (
+                    '[tox]\nenvlist = py36\n\n'
+                    '[isort]\nknown_third_party=cfgv\n'
+                ),
+            },
+        ),
+    ],
+)
+def test_integration_no_section(
+    tmpdir,
```

Comment:
please 2-indent function parameters to differentiate from code below (yes I should write a formatter for this so I don't have to complain)

Reply:
Sure